src/vma/util/asm-x86.h

```diff
@@ -36,6 +36,16 @@
 	dst += 8;			\
 	src += 8
 
+#define COPY_64B_NT_C(dst, src)	\
```

Comment:
@serglelv can we use something like https://github.com/efficient/rdma_bench/blob/master/drivers/libmlx5-1.0.2mlnx1/src/doorbell.h#L73
to detect KVM case w/o introducing new configure option?